bookkeeper-server/pom.xml

```diff
@@ -263,31 +290,6 @@
         </dependency>
       </dependencies>
     </profile>
-    <profile>
-      <id>tls-certs</id>
-      <build>
```

Comment:
It seems to me that you are dropping this, so openssl will be executed at every run

Reply:
Yes. Previously I created a long lived self signed certificate which lacked host name verification. With the current changes, certificate verification requires the host ip and hostname to be part of the certificate. So, each host building/running tests would need to generate host specific self signed certificates.

Reply:
I prefer to create certs only when needed, with this setup every time you launch any single  test from the command line you will create new certificates, it slows down every day work, for tests and code which is not changed very frequently

Cc @ivankelly @sijie

Reply:
an alternative is to move tls related stuff to its module, so the certs generation can only happen in that module. but that's probably in a separate PR.

@eolivelli if not regenerate certs, how are you going to test certificate verification as what @kishorekasi suggested?

Reply:
@sijie my suggestion it to generate certificates only when running tests which need the certificates, otherwise you will end generating certs at every mvn test -Dtest=xxxx

We can do it in a @beforeclass or another approach is to  add some logic in the bash script so that if the cert is already present it won't be regenerated

Reply:
beforeclass sounds doable to me.

Reply:
we can do it in a subsequent change if this change is ready to go.

Reply:
I can accept this patch as is, but please  submit a follow up patch to introduce creation in BeforeClass .